### PR TITLE
[WEEK 9] Project Shopping Cart - Create project template

### PR DESCRIPTION
**Pull request by jeanpsv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
-# sd-01-week9-project-shopping-cart
-Repositório para o aluno enviar o projeto da semana 9 co curso. Tema: Web Storage e JavaScript Promises
+# Boas vindas ao repositório do projeto de _animações_ e _mobile first_!
```
Comment (LucasCFerraz) [inline README.md]: Seems wrong title 🗡

**Comment by antoniosb** [comment]
@jeanpsv checkout the docs page on [how to setup the project repo](https://github.com/betrybe/course/blob/master/docs/projects/repo_config.md)

**Comment by jeanpsv** [comment]
> @jeanpsv remember that the `.travis.yml` script still have to be tuned based on the project requirements.

Yes, I've just copied the template, but I have to research to create one for this project

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  7
```
Comment (antoniosb) [inline .travis.yml]:
```suggestion
  12
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,256 @@
...
+
+Ao clicar no **produto no carrinho de compra**, ele deve ser removido da lista.
+Para isso, uma função (já existente) chamada `cartItemClickListener(event)` deve ser implementada com a lógica necessária para realizar a remoção.
+
+### 7. Salve o carrinho de compras no **LocalStorage**
+
+O carrinho de compras deve ser salvo no **LocalStorage**, ou seja, todas as **adições** e **remoções** devem ser abordadas para que a lista atual seja salva.
+
+O carrinho de compras deve ser populado ao carregar a página.
+
```
Comment (antoniosb) [inline README.md]: We should add a requisite that states we will only assess the project if there no issues on CodeClimate and TravisCI. What do you think ?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,256 @@
...
+## Utilização dos templates (index.html, style.css e script.js)
+
+Você **não** deve alterar a lógica das funções implementadas no arquivo `script.js`.
+
+Os arquivos `index.html` e `style.css` serão alterados apenas se você decidir fazer os requisitos bônus.
+Você pode alterar estes arquivos, mas lembre-se de **não** alterar a hierarquia presente na estrutura.
+
+## Requisitos do projeto
+
```
Comment (antoniosb) [inline README.md]:
```suggestion

⚠️ Lembre-se que o seu projeto só será avaliado se estiver passando pelos _checks_ do **CodeClimate** e do **TravisCI**

```

File: README.md (hunk 3 above)
Comment (antoniosb) [reply README.md]: added a suggestion above ☝️

**Pull request merged**